workers/unity/Assets/Plugins/Improbable/Sdk/Src/Editor/Editor/.iOSBuildTools/XcodeSettingsBuilder.cs

```diff
@@ -0,0 +1,103 @@
+
+using System.IO;
+using UnityEditor;
+using UnityEditor.iOS.Xcode;
+using UnityEngine;
+
+namespace Improbable.Editor.Addons.iOSBuildTools
```

Comment:
completely didn't see that this still exists. @jared-improbable we either want to add either update that file in the repo or remove it. The current version does not work correctly, I believe.

Not really relevant for this PR though

Reply:
I've confirmed that this is superfluous but harmless. I've raised an internal ticket to remove it in future versions.